core/src/main/resources/reference.conf

```diff
@@ -209,6 +209,37 @@ engine {
   }
 }
 
+frontend {
+  default = "WDL"
+  providers {
+    WDL {
+      supported-versions = [
+        {
+          workflow-type = [ "WDL" ],
+          workflow-type-version = [ ] # actually require this to be blank as assent to Cromwell's disregard for WDL versions?
```

Comment:
We should just assign some arbitrary version to pre-version WDL, e.g. `draft-2`, which is technically what we say it is. After that if we make clear that draft versions are moving targets I think it'll be fine.